src/components/Editor/Invitees/InviteesListSearch.vue

```diff
@@ -237,6 +240,30 @@ export default {
 				}
 			})
 		},
+		async findAttendeesFromCircles(query) {
+			let results
+			try {
+				results = await circleSearchByName(query)
+			} catch (error) {
+				console.debug(error)
+				return []
+			}
+
+			return results.filter((circle) => {
+				return true
+			}).map((circle) => {
+				return {
+					commonName: circle.displayname,
+					calendarUserType: circle.calendarUserType,
+					email: circle.email,
```

Comment:
so conceptually it fetches the circle, expands to the individuals and adds everyone with an email address as attendee, right?

Reply:
Have to figure it out too 😂